odoo/http.py

```diff
@@ -1522,6 +1522,10 @@ def get_db_router(self, db):
             return self.nodb_routing_map
         return request.registry['ir.http'].routing_map()
 
+
+def redirect(location, code=302, Response=Response):
+    return werkzeug.utils.redirect(location, code=code, Response=Response)
```

Comment:
any reason to put it so far from the other redirection methods in this file? I'd put it above `local_redirect()` and `redirect_with_hash`, perhaps? :thinking:

Reply:
good question :/ why I choose this place ... :thinking: 

Moved with other redirects

Reply:
Yes, now, I remember...

Because I need to be sure that Class Response is already defined.
But we can probably remove this parameters...

and replace

`def redirect(location, code=302, Response=Response):`
by 
`def redirect(location, code=302):`

=> btw moved in ir.http now